packages/forms/src/types.js

```diff
@@ -26,8 +27,12 @@ type TableSchema = {
 
 type Schema = Array<TableSchema>;
 
-type SchemaContextValue = {
-  schema: Schema,
+type SchemaContextValue = Schema;
+
+type RenderableProps = {
+  component?: React.ComponentType<*>,
+  children?: ((props: Object) => React.Node) | React.Node,
+  render?: (props: Object) => React.Node
```

Comment:
Replace `React.Node` to `React$Node`. Then you can escape `import * as React`.

Reply:
@ej9x fixed